usagov/vote-gov
# Vote-1642 -> accessibility updates

[Vote-1642](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1642&assignee=611272e0eb7aef00694b8a97)

This Pr will add accessibility updates to new languages 

to test:

1. pull down branch 
2. review files and ensure that updates have been made from [accessibility doc](https://docs.google.com/spreadsheets/d/198F9Cvh4M0qVaz5YY4hEkCSHiM8tWpfc/edit#gid=1447636670)
3. also visit `localhost:1313` and inspect to ensure text is present

> **Note**
> Please check on the Arabic translation that if you inspect, you can see all elements on the page. for some reason for me when I do that it all goes away and I don't know if its a me issue or not...
